keyboards/keebio/levinson/keymaps/omgvee/keymap.c

```diff
@@ -0,0 +1,205 @@
+  //|----+----+----+----+----+----|    |----+----+----+----+----+----|
+      X3 ,LCTL,LALT,LGUI, LOW, ENT,      SPC,RAIS,RALT,    ,    ,
+  //`----+----+----+----+----+----'    `----+----+----+----+----+----'
+  ),
+
+  [_DVORAK] = LAYOUT_kc(
+  //,----+----+----+----+----+----.    ,----+----+----+----+----+----.  
+     TAB ,QUOT,COMM,DOT , P  , Y  ,      F  , G  , C  , R  , L  ,MINS, 
+  //|----+----+----+----+----+----|    |----+----+----+----+----+----|
+      X0 , A  , O  , E  , U  , I  ,      D  , H  , T  , N  , S  ,SLSH,
+  //|----+----+----+----+----+----|    |----+----+----+----+----+----|
+     LSFT,SCLN, Q  , J  , K  , X  ,      B  , M  , W  , V  , Z  ,SENT,
+  //|----+----+----+----+----+----|    |----+----+----+----+----+----|
+      X3 ,LCTL,LALT,LGUI, LOW, ENT,      SPC,RAIS,RALT,    ,    ,
+  //`----+----+----+----+----+----'    `----+----+----+----+----+----'
+  ),
+
+  [_FN3] = LAYOUT_kc(
+  //,----+----+----+----+----+----.    ,----+----+----+----+----+----.
+     F12 , F1 , F2 , F3 , F4 , F5 ,      F6 , F7 , F8 , F9 ,F10 ,F11 ,
+  //|----+----+----+----+----+----|    |----+----+----+----+----+----|
+         ,    ,    ,    ,    ,    ,     MS_L,MS_D,MS_U,MS_R,WH_L,WH_R,
+  //|----+----+----+----+----+----|    |----+----+----+----+----+----|
+         ,    ,    ,    ,    ,    ,     BTN1,WH_D,WH_U,BTN2,    ,    ,
+  //|----+----+----+----+----+----|    |----+----+----+----+----+----|
+         ,    ,    ,    ,    ,    ,         ,    ,    ,    ,    ,
+  //`----+----+----+----+----+----'    `----+----+----+----+----+----'
+  ),
+
+/* Adjust (Lower + Raise)
+ * ,-----------------------------------------------------------------------------------.
+ * |      | Reset|RGB Tg|RGB Md|Hue Up|Hue Dn|Sat Up|Sat Dn|Val Up|Val Dn|      |      |
+ * |------+------+------+------+------+-------------+------+------+------+------+------|
+ * |      |      |      |Aud on|Audoff|AGnorm|AGswap|Qwerty|Colemk|Dvorak|      |      |
+ * |------+------+------+------+------+------|------+------+------+------+------+------|
+ * |      |      |      |      |      |      |      |      |      |      |      |      |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |      |      |      |      |      |             |      |      |      |      |      |
+ * `-----------------------------------------------------------------------------------'
+ */
+  [_ADJUST] = LAYOUT( \
+      RESET,  QWERTY, COLEMAK,  DVORAK, _______, EEP_RST,   KC_MNXT, KC_MPRV, KC_MFFD, KC_MRWD,  DVORAK, KC_EJCT, \
+    RGB_TOG, RGB_HUI, RGB_SAI, RGB_VAI, KC_PGUP, KC_HOME,   KC_LEFT, KC_DOWN,   KC_UP, KC_RGHT,   CK_UP, CK_DOWN, \
+    RGB_MOD, RGB_HUD, RGB_SAD, RGB_VAD, KC_PGDN,  KC_END,   KC_BRID, KC_BRIU, KC_VOLD, KC_VOLU, KC_MUTE, KC_MPLY, \
+    BL_TOGG,  BL_INC,  BL_DEC, BL_BRTG, _______, _______,    CK_RST, _______, CK_TOGG,  MU_MOD,  MU_TOG,  AU_TOG \
+  )
+
+
+};
+
+#ifdef AUDIO_ENABLE
```

Comment:
This block should be removed.

Reply:
did you mean the #ifdef AUDIO_ENABLE until it ends? or did you mean the `persistent_default_layer_set` function definition? the latter I am committing the change as we speak

Reply:
Specifically, all of the `#ifdef AUDIO_ENABLE` blocks.  

The newer function handles that with a define, so it makes these `PLAY_SONG` function calls redundant:
https://docs.qmk.fm/#/ref_functions?id=setting-the-persistent-default-layer

Reply:
Sorry, I meant to open a PR to your branch, but github decided a commit was a better idea :(

Reply:
oh wow, sweet. Nah, it's no problem for me, in fact, it saved me a bit of cherrypicking on my work box, since I had made the changes and PR from my home box :) So, thank you tons actually!

and again, thanks a bunch for the AUDIO stuff. Trying to keep up with the changes and particularly deprecated things in old keymaps of other folks... Perhaps it'd be best if to prune these stale people of code from everywhere at some point... let me see what I can find